WhaleSpotting/react-app/src/styles/ReportSighting.scss

```diff
@@ -0,0 +1,90 @@
+@import "./Constants.scss";
+.report-sighting{
+    font-family: $fontFamily;
+
+    .banner-image{
+        height: 40vh;
+        object-fit: cover;
+        width: 100%;
+    }
+
+    .required{
+        color: $secondaryDark;
+        font-size: $mobileBodyFontSize;
+    }
+
+    .container{
+        margin: auto;
+        max-width: 800px;
+        padding: 4% 5%;
+    }
+
+    .card-component{
+        background: $primaryCard;
+        border-radius: 5px;
+    }
+
+    .title{
+        text-align: center;
+        font-size: $titleFontSize;
+    }
+
+    .sighting-details{
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin-top: 6%;
+        padding: 2% 5% 5% 5%;
+    }
+
+    .input-box{
+        margin: 3% 0;
+        width: 46%;
+    
+        .input-field{
+            height: 85%;
+            width: 100%;
+            padding-left: 10px;
+            border-radius: 5px;
+            transition: all 0.3s ease;
+            border: 1px solid $primaryColour;
+            font-size: $bodyFontSize;
+            
+            &:focus{
+                outline: none;
```

Comment:
Not sure this is doing anything?

Reply:
The font-sizing is giving the font it's size otherwise it is smaller then we want. 
The focus outline: none is required, if we don't have it then a border out on the input box is displayed which we don't want.